woocommerce/woocommerce-ios
# [Custom Amounts M3] Sync Order Details with Custom Amounts changes

Closes: #11105


## Description

With this PR we sync the order details with custom amounts (fees) changes after editing them. In order to do that, we create a results controller in the same way it's done for other order components. Because we were not doing this before, we didn't show any change in the order custom amounts' name.

## Testing instructions


1. Go to Orders
2. Tap on + to create a new order
3. Add a custom amount to the order. Add a name to it.
4. Tap on Done to create the order and open the order details screen. 
5. Tap on Edit.
6. Change the name of the custom amount.
7. Tap to save it and tap on Done to go back to order details.
8. See that the name is changed in the order details screen.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/1864060/d9529850-7ae5-4262-8218-9885217a357f





---
